apps/www/components/CallToActions/useCallToAction.ts

```diff
@@ -9,21 +8,20 @@ import { useCallToActionsQuery } from './graphql/useCallToActionsQuery'
  *  data: CallToAction
  *  loading: boolean
  *  error: ApolloError
- *  handleDismiss: (id: string) => void
  * }
  */
 export default function useCallToAction(): {
-  data?: CallToAction
+  data?: CallToAction[]
   loading: boolean
   error?: ApolloError
   refetch: () => void
 } {
   const { data, loading, error, refetch } = useCallToActionsQuery()
   const callToActions = data?.me?.callToActions
-  const activeCTA = callToActions?.find((cta) => !cta.acknowledgedAt)
+  const activeCTAs = callToActions?.filter((cta) => !cta.acknowledgedAt)
```

Comment:
Just double-checking here:

With this change, you might see many call to actions at once. Thus far intentional idea was to render only one call to action at a time. (If acknowledged, next would rendered and so forth.)

Is that wanted?

Reply:
Yes/no, it wasn't clear from the code that this was intentional, since we always fetch all of them (even acknowledged ones). So maybe the new and improved query should be changed to always return _one_ active CTA?

Reply:
Sounds to me like a (business) logic question.

Since we've not an actual set of rules, I'd opt for the next best thing: Performance.

It's very likely cheaper to return all CTAs at once than acknowledging one, fetch next and display it?

(Other approach might be to alter API to return next call to action upon acknowledging. Which might feel counter-intuitive: You mutate using a CTA ID but receive a new one or none?)

Reply:
(Settled to not return one active CTA atm.)